web-registry/src/pages/ProjectReview/ProjectReview.tsx

```diff
@@ -66,7 +66,7 @@ export const ProjectReview: React.FC = () => {
     wallet,
     error,
     setError,
-    setDeliverTxResponse,
+    // setDeliverTxResponse,
```

Comment:
commented this out - assuming we don't want to use. In the long run, I think we'd probably want to `useMsgClient` directly in the context route